components/perf-tools/tau/SPECS/tau.spec

```diff
@@ -236,6 +236,9 @@ done
 %install
 mkdir -p %{buildroot}/%{_docdir}
 
+# Remove the pre-built JOGL libraries which depend on Nvidia CG Toolkit
+find TAUBUILD/ -name "libjogl_cg.so" -delete
```

Comment:
Can't you just specify the complete path without using find. With find more file might be deleted than what we want.

Reply:
In TAU's sources this file comes from `tools/src/contrib/jogl/$ARCH/libjogl_cg.so.gz`. Depending on the current architecture it is being uncompressed and put in a temp folder. I haven't tracked whether the .so file is being copied/moved elsewhere after that.
I think all occurrences of this file should be deleted. I am not sure which one(s) are seen and processed by `find-requires`.